features/login/impl/src/main/kotlin/io/element/android/features/login/impl/screens/createaccount/WebViewMessageInterceptor.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2023, 2024 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ * Please see LICENSE in the repository root for full details.
+ */
+
+package io.element.android.features.login.impl.screens.createaccount
+
+import android.graphics.Bitmap
+import android.webkit.JavascriptInterface
+import android.webkit.WebResourceRequest
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import androidx.webkit.WebViewCompat
+import androidx.webkit.WebViewFeature
+
+class WebViewMessageInterceptor(
```

Comment:
Definitely not for now, but maybe we should extract this to some shared module (androidutils?).

Reply:
Yes, I had though about it, but since this is supposed to be a temporary code...